### DISCUSSION TITLE
MR Software is closing - future of their Controlling, Vacation, Sickness plugins

### DISCUSSION THREAD

**Discussion by davidmehren** [kimai/kimai#4145] [General] [answered]
MR Software, developer of a bunch of Kimai plugins, is [in liquidation](https://mrsoft.gmbh/) and closes today.

I'm wondering what that means for all their plugins. We planned to use them, but no plans for compatibility with Kimai 2 were announced (and they didn't give a clear answer when I inquired via email), so we hesitated.

The plugins should probably be removed from the store? @kevinpapst

**Comment by kevinpapst** [comment, marked as answer]
Read this blog entry: https://www.kimai.org/blog/2023/controlling-plugin-mr-software/

There is a plugin that replaces the MR logic: https://www.kimai.org/store/controlling.html
This page contains an introduction vide as well.

You can contact me via support@kimai.org if you have questions which are NOT answered in the linked resources.
